.github/workflows/project-meta-sync.yml

```
@@ -0,0 +1,104 @@
+              chore/*|build/*) TYPE_FIELD="Task";;
+            esac
+          fi
+
+          # GraphQL update of fields
+          cat >mutation.graphql <<'EOM'
+          mutation($project: ID!, $item: ID!, $statusId: String!, $priorityId: String!, $typeId: String) {
+            updateProjectV2ItemFieldValue(input: {projectId: $project, itemId: $item, fieldId: "STATUS_FIELD_ID", value: { singleSelectOptionId: $statusId } }) { clientMutationId }
+            updateProjectV2ItemFieldValue(input: {projectId: $project, itemId: $item, fieldId: "PRIORITY_FIELD_ID", value: { singleSelectOptionId: $priorityId } }) { clientMutationId }
+            updateProjectV2ItemFieldValue(input: {projectId: $project, itemId: $item, fieldId: "TYPE_FIELD_ID", value: { text: $typeId } }) { clientMutationId }
+          }
+EOM
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 103: could not find expected ':'

(syntax-check)